slangpy/tests/slangpy_tests/helpers.py

```diff
@@ -121,10 +131,39 @@ def get_device(
     return device
 
 
-# Helper that creates a module from source (if not already loaded) and returns
-# the corresponding slangpy module.
+TORCH_DEVICES: dict[str, Device] = {}
+
+
+# Helper that gets a device that wraps the current torch cuda context.
+# This is useful for testing the torch integration.
+def get_torch_device(type: DeviceType) -> Device:
+    import torch
 
+    # For testing, comment this in to disable backwards passes running on other threads
+    torch.autograd.grad_mode.set_multithreading_enabled(False)
```

Comment:
Does torch integration work if `autograd` uses multithreading? If not, it should probably be disabled in `slangpy.core.utils.create_torch_device` as well.